android/src/main/java/com/th3rdwave/safeareacontext/SafeAreaUtils.kt

```diff
@@ -72,10 +71,18 @@ fun getSafeAreaInsets(view: View): EdgeInsets? {
   val visibleRect = android.graphics.Rect()
   view.getGlobalVisibleRect(visibleRect)
   return EdgeInsets(
-      top = max(windowInsets.top - visibleRect.top, 0f),
-      right = max(min(visibleRect.left + view.width - windowWidth, 0f) + windowInsets.right, 0f),
-      bottom = max(min(visibleRect.top + view.height - windowHeight, 0f) + windowInsets.bottom, 0f),
-      left = max(windowInsets.left - visibleRect.left, 0f))
+      top = maxOf(windowInsets.top - visibleRect.top, windowInsets.top, 0f),
+      right =
+          maxOf(
+              min(visibleRect.left + view.width - windowWidth, 0f) + windowInsets.right,
+              windowInsets.right,
```

Comment:
```suggestion
              min(visibleRect.left + view.width - windowWidth, 0f) + windowInsets.right,
```

Isn't this the same thing, since it's a `min(something, 0) + windowInsets.right`

Reply:
It's not the same - as I was testing it, the value in `min` was returning negative value (-66, actually) 🤔